src/chainNode.ts

```diff
@@ -44,4 +62,57 @@ export default class ChainNode {
     const members = (await this.api.query.membership.members()).toJSON()
     return listParser.parse(members)
   }
+
+  async getAttachmentApiAddresses(account: string): Promise<OrgData[]> {
```

Comment:
is there a reson this returns an array. It seems we only ever return one element